firefox_puppeteer/errors.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.errors import *
```

Comment:
I know but I do not see another option here to easily extend the Marionette errors. I would like that our tests only have to import this module and have access to all available exception classes. Do you have any other option in mind?